src/App.tsx

```diff
@@ -52,7 +62,26 @@ function App() {
           as="textarea"
           value={textarea}
           placeholder={defaultText}
+          ref={inputRef}
+          onKeyDown={(e: any)=> {
```

Comment:
It's safer to not use the `any` type here, this should be of type `React.KeyboardEvent<HTMLTextAreaElement>`